### DISCUSSION TITLE
Change (background) color on hover

### DISCUSSION THREAD

**Discussion by phuhl** [polybar/polybar#2694] [Ideas]
I saw, we have cursor change on hover, but I think, being able to change the color on hover would be even more sick and add a lot more live-ness to the bar.

**Comment by patrick96** [comment]
That would certainly be cool, but I don't see this being added to polybar. Polybar has nowhere near the required functionality to enable something like this.

**Comment by AllergicMushroom** [comment]
I am also interested in this. Just what is polybar missing for this?
